rust/origen/src/utility/file_actions.rs

```diff
@@ -10,6 +10,7 @@ enum Operation {
     InsertBefore,
     InsertBeforeAll,
     InsertAfter,
+    #[allow(dead_code)]
```

Comment:
Is this needed? I don't see any other changes in this file that would need this.

Reply:
Yeah definitely needed to fix a build warning, I just checked master and the warning is there too, so that's why there is no other corresponding change here